release.yaml

```diff
@@ -51,3 +51,7 @@ rancher-webhook:
   - 2.0.7+up0.3.7-rc2
 harvester-cloud-provider:
   - 102.0.2+up0.2.3
+ui-plugin-operator:
```

Comment:
This is confusing although it is right:
https://github.com/rancher/ui-plugin-operator/blob/main/charts/ui-plugin-operator/Chart.yaml

102.0.3 is right.
+up0.2.1 = version instead of appVersion from the upstream chart